docker-compose.yml

```diff
@@ -33,7 +33,8 @@ services:
       - db
 
   portalbackend:
-    image: hbpmip/portal-backend:5.0.2
+    #image: hbpmip/portal-backend:5.0.5
+    image: jerrypan44/portal-backend:latest
```

Comment:
we need a published image with a release tag under hbpmip/portal-backend:5.1.0

Reply:
Shouldn't we do this after we merge?

We usually merge the code if it looks good and then publish the images. What do you think?

Reply:
Do you just want to have it like that to be ready for the post-merge images?